_posts/2017-05-05-Striped.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Striped
+date: 2017-05-05
+homepage: https://github.com/aroaminggeek/Jekyll-HTML5Up-Striped
+download: https://github.com/muan/scribble/archive/master.zip
+demo: http://scribble.muan.co/2013/05/06/scribble-the-jekyll-theme/
+author: A Roaming Geek
+thumbnail: striped.png
+license: CCA 3.0
```

Comment:
Please change license to it's proper name `Creative Commons Attribution 3.0 Unported License`